src/lerner_lab_to_nwb/seiler_2024/seiler_2024fiberphotometryinterface.py

```diff
@@ -254,46 +371,56 @@ def add_to_nwbfile(
             description="The fluorescence from the blue light excitation (465nm) corresponding to the calcium signal in the DMS.",
             data=H5DataIO(tdt_photometry.streams["Dv1A"].data, compression=True),
             unit="a.u.",
-            fibers=dms_fiber_ref,
-            excitation_sources=dms_excitation_ref,
-            photodetectors=photodetector_ref,
-            fluorophores=dms_fluorophore_ref,
             rate=tdt_photometry.streams["Dv1A"].fs,
+            fiber_photometry_table_region=dms_signal_region,
         )
         dms_reference_series = FiberPhotometryResponseSeries(
             name="dms_reference",
             description="The fluorescence from the UV light excitation (405nm) corresponding to the isosbestic reference in the DMS.",
             data=H5DataIO(tdt_photometry.streams["Dv2A"].data, compression=True),
             unit="a.u.",
-            fibers=dms_fiber_ref,
-            excitation_sources=dms_excitation_ref,
-            photodetectors=photodetector_ref,
-            fluorophores=dms_fluorophore_ref,
             rate=tdt_photometry.streams["Dv2A"].fs,
+            fiber_photometry_table_region=dms_reference_region,
         )
         dls_signal_series = FiberPhotometryResponseSeries(
             name="dls_signal",
             description="The fluorescence from the blue light excitation (465nm) corresponding to the calcium signal in the DLS.",
             data=H5DataIO(tdt_photometry.streams["Dv3B"].data, compression=True),
             unit="a.u.",
-            fibers=dls_fiber_ref,
-            excitation_sources=dls_excitation_ref,
-            photodetectors=photodetector_ref,
-            fluorophores=dls_fluorophore_ref,
             rate=tdt_photometry.streams["Dv3B"].fs,
+            fiber_photometry_table_region=dls_signal_region,
         )
         dls_reference_series = FiberPhotometryResponseSeries(
             name="dls_reference",
             description="The fluorescence from the UV light excitation (405nm) corresponding to the isosbestic reference in the DLS.",
             data=H5DataIO(tdt_photometry.streams["Dv4B"].data, compression=True),
             unit="a.u.",
-            fibers=dls_fiber_ref,
-            excitation_sources=dls_excitation_ref,
-            photodetectors=photodetector_ref,
-            fluorophores=dls_fluorophore_ref,
             rate=tdt_photometry.streams["Dv4B"].fs,
+            fiber_photometry_table_region=dls_reference_region,
         )
```

Comment:
Is there a reason why you need 4 different FiberPhotometrySeries?
If the have the same rate and the same starting time, you could just create one with a single table region with the appropriate ordered indices of FibePhotometryTable's row